ui/src/components/ModalWrapper/index.tsx

```diff
@@ -30,13 +32,16 @@ export const ModalWrapper: FC<ModalWrapperProps> = ({
       <div className="flex max-h-[70vh] flex-col gap-5 overflow-y-auto p-[1px]">
         {children}
       </div>
-      {showSaveBtn && (
-        <DialogFooter>
+      <DialogFooter>
+        <Button type="button" onClick={onCancel}>
```

Comment:
Should we use the ghost variant for the cancel button, as we do for all other modals?

![CleanShot 2024-02-06 at 11 52 51@2x](https://github.com/direktiv/direktiv/assets/121789579/82566f59-aa13-44e0-bf0c-9f0418a6a163)

`<Button variant="ghost" type="button" onClick={onCancel}>`

Reply:
Good point. Done in 5443bace48e2ebdeae5f823569e23f1cbe6002d5